### PR TITLE
options to recognize spec/srpm files (RhBug:1241135)

### PR DESCRIPTION
**Pull request by MichaelMraka**

### CODE REVIEW

**Comment by rholy** [comment]
Can we have the new options mutually exclusive?

File: plugins/builddep.py [hunk 1]
```diff
@@ -157,7 +166,10 @@ def _src_deps(self, src_fn):
                 logger.error("Error: public key not available, add "
                              "'--nogpgcheck' option to ignore package sign")
                 return
-            raise
+            elif e[0] == 'error reading package header':
```
Comment (rholy) [inline plugins/builddep.py]: have you bothered testing this?
(https://bugzilla.redhat.com/show_bug.cgi?id=1218299)

**Comment by rholy** [comment]
https://github.com/rpm-software-management/dnf-plugins-core/pull/103

**Comment by MichaelMraka** [comment]
Merged.

**Pull request closed without merging**